elastic/kibana
# [Upgrade Assistant] Add warning when remote clusters are configured

## Summary

Fixes https://github.com/elastic/kibana/issues/124465

This PR adds a warning to the ES deprecations page if we detect the user has any remote clusters configured. It also adds a locator to the `remote_clusters` plugin, so that we can easily link to Remote Clusters from UA.

<img width="1005" alt="Screen Shot 2022-02-09 at 2 03 41 PM" src="https://user-images.githubusercontent.com/5226211/153272613-9cb45882-ffc2-4371-9dd2-83e056e6353c.png">


### How to test
1. Navigate to Upgrade Assistant --> ES deprecations. Verify no warning message displays.
2. Create a remote cluster(s). (See [CCR readme in Kibana](https://github.com/elastic/kibana/tree/main/x-pack/plugins/cross_cluster_replication#quick-steps-for-testing-cross-cluster-replication) if you need help)
3. Navigate to Upgrade Assistant --> ES deprecations. Verify warning message displays and link navigates to remote clusters UI.
